src/components/SubjectLoading.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const SubjectLoading = ({ loaded }) => {
+  const transform = !loaded ? 'scale(0.35)' : '';
```

Comment:
This prop was added because there is an offset with how the parent's `<g>` transform changes once a subject is loaded. Look at the `SVGImage`'s parent `<g>` in the `SubjectViewer` and you'll notice there is a transform/scale that changes when `RESET_VIEW` is called in the `SVGImage`'s `onLoad` function. This shouldn't really be an issue since a loaded image will make the spinner disappear, but it was mainly added just in case.